src/time_entry.cc

```diff
@@ -457,8 +457,8 @@ void TimeEntry::LoadFromJSON(Json::Value data) {
     SetBillable(data["billable"].asBool());
     SetDurOnly(data["duronly"].asBool());
     SetUpdatedAtString(data["at"].asString());
+    SetUIModifiedAt(Formatter::Parse8601(data["at"].asString()));
```

Comment:
@IndrekV This change broke library logic. If `ui_modified_at` is not  `0` it is marked as unsynced and `BaseModel::NeedsPUT()` returns `true`.

Reply:
Good call, I'll look for a fix.

Reply:
I've submitted a PR for this #3346, you can check out if it looks good

Reply:
this should fix it b5a5f15b5053094d20293aa16d33adf94111bd42